### ISSUE TITLE
icon files are not copied to dist folder in build

### ISSUE THREAD

**Issue by maxisam** [angular/angular-cli#8120]
### Bug Report or Feature Request (mark with an `x`)
```
- [X ] bug report -> please search issues before submitting
- [ ] feature request
```

### Versions.
v1.4.8

### Repro steps.
import fontawesome in style.scss like

`@import "~font-awesome/css/font-awesome.css";`

### Desired functionality.

fontawesome-webfont files should be in dist folder.

### Mention any other details that might be useful.

It works on v1.4.7

**Comment by jcyovera** [comment]
Update to version 1.4.9 it works for me.

**Comment by maxisam** [comment]
I can confirm it is fixed by 1.4.9
